crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -52,6 +54,22 @@ def knife_files
           file_path = new_file
         end
 
+        if file.basename.to_s =~ /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          # override with migrated attributes
+          json["attributes"]["network"] = SchemaMigration.migrated_attributes("network", json)
+          Tempfile.open("network") do |f|
+            f.write JSON.pretty_generate(attributes: json["attributes"])
+            cmd = [
+              "sudo",
+              "cp",
+              f.path,
+              "/etc/crowbar/network.json"
+            ]
+            system(*cmd)
```

Comment:
Confirmed. If I put ```f.close``` before the cmd the file in not empty anymore. But I am sure there's a more correct way to address this.